README.md

```diff
@@ -37,36 +41,44 @@ cd eva-sub-cli
 python -m pip install .
 ```
 
-To check it is installed correctly, you can run:
+To verify that the cli tool is installed correctly, run the following command, and you should see the help message displayed : 
 ```bash
 eva-sub-cli.py -h
 ```
 
-### From source natively
+### 3. From source natively
 
-This method requires the following:
+This installation method requires the following :
 * Python 3.8+
 * [Nextflow](https://www.nextflow.io/docs/latest/getstarted.html) 21.10+
 * [biovalidator](https://github.com/elixir-europe/biovalidator) 2.1.0+
 * [vcf-validator](https://github.com/EBIvariation/vcf-validator) 0.9.6+
 
-Install each of these and ensure they are available on the path.
-Then git clone the repo or install the newest release as described above.
+Install each of these and ensure they are included in your PATH. Then, either clone the repository using Git or install the latest release as previously described.
+
+## Getting started with the eva-sub-cli tool 
 
-## Input files for the validation and submission tool
 
-There are two ways of specifying the VCF files and associated reference genome
+The [README file ](Getting_Started_with_eva_sub_cli.md) serves as an introductory guide for users of the eva-sub-cli tool. It includes instructions on how to prepare your data and metadata, ensuring that users are equipped with the necessary information to successfully submit variant data. This guide is essential for new users, offering practical advice and tips for a smooth onboarding experience with the eva-sub-cli tool.
 
-### Using  `--vcf_files` and `--reference_fasta`
+## eva-sub-cli tool: Options and parameters guide
 
-This allows you to provide multiple VCF files to validate and a single associated reference genome file.
-The VCF files and the associated reference genome file must use the same chromosome naming convention 
+The eva-sub-cli tool provides several options/parameters that you can use to tailor its functionality to your needs. Understanding these parameters is crucial for configuring the tool correctly. Below is an overview of the key parameters and options:
 
-### Using metadata file by providing `--metadata_json` or `--metadata_xlsx`
+| OPTIONS/PARAMETERS         | DESCRIPTION                                                                                                                                                                                                        |
+|----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --version                  | Shows version number of the program and exit                                                                                                                                                                       |
```

Comment:
Technically I've edited the `--help` text in the code... we can transfer those edits here but it's not a big deal.